src/index.js

```diff
@@ -0,0 +1,67 @@
+import './style/index.css';
+
+const toDoContainer = document.querySelector('.to-do-list--container');
+const toData = [
+  {
+    description: 'Adding a new item',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'Adding a new item',
+    completed: false,
+    index: 2,
+  },
+  {
+    description: 'Adding a new item',
+    completed: false,
+    index: 3,
+  },
+  {
+    description: 'Adding a new item',
+    completed: false,
+    index: 4,
+  },
+  {
+    description: 'Adding a new item',
+    completed: false,
+    index: 5,
+  },
+  {
+    description: 'Adding a new item',
+    completed: false,
+    index: 6,
+  },
+];
+
+const renderToDoList = () => {
+  const markUp = toData
+    .map((toDo) => `
```

Comment:
- [ ] Hello great developer! Awesome implementing most of the project requirement but kindly ensure that the list is ordered in accordance to the index values for each task. 👍 

<img width="784" alt="Untitled" src="https://github.com/Yonashenok/To-Do-list/assets/103386183/2e27d693-0e16-4b20-a361-04dffd0c854d">

Reply:
hey it in the order that way i put the index of the in the list